infra/package.json

```diff
@@ -11,6 +11,8 @@
   },
   "license": "MIT",
   "devDependencies": {
+    "@babel/plugin-transform-modules-commonjs": "7.23.3",
+    "@nx/webpack": "17.2.8",
```

Comment:
We are currently using NX v16, the NX packages should follow suit. If this package is required then we should attempt to upgrade NX first in a separate PR.

Reply:
This is in the `infra` folder, which is disconnected from the main NX and `package.json`. I think the packages are required; CI was complaining about these packages missing, but not locally :shrug:

Reply:
Setting like so:
```suggestion
    "@nx/webpack": "@16.9.0",
```